py/setup.py

```diff
@@ -0,0 +1,89 @@
+from setuptools import setup, Extension
+from setuptools.command.build_ext import build_ext
+from os.path import expanduser
+from glob import glob
+from itertools import chain
+import sys
+import setuptools
+
+__version__ = '0.0.1'
+
+
+class get_pybind_include(object):
+    """Helper class to determine the pybind11 include path
+    The purpose of this class is to postpone importing pybind11
+    until it is actually installed, so that the ``get_include()``
+    method can be invoked. """
+
+    def __init__(self, user=False):
+        self.user = user
+
+    def __str__(self):
+        import pybind11
+        return pybind11.get_include(self.user)
+
+
+sources = list(chain(
+    glob('*.cpp'),
+    glob('../replayer/*.cpp'),
+    glob('../client/*.cpp'),
+))
+print(sources)
+
+ext_modules = [
+    Extension(
+        'torchcraft',
+        sources,
+        include_dirs=[
+            # Path to pybind11 headers
+            get_pybind_include(),
+            get_pybind_include(user=True),
+            "../include",
+            "../",
+            # TODO Dynamically search for this somehow???
+            expanduser("~/torch/install/include"),
```

Comment:
Doesn't Torch set some variables when activated? We can probably get shell env and run `os.expandpath` or something like that.